apps/condo/domains/acquiring/utils/handlers.js

```diff
@@ -0,0 +1,74 @@
+const { get, isEmpty } = require('lodash')
+
+const conf = require('@core/config')
+
+const { COUNTRIES, DEFAULT_LOCALE } = require('@condo/domains/common/constants/countries')
+
+const { PAYMENT_ERROR_STATUS } = require('@condo/domains/acquiring/constants/payment')
+
+const { sendMessage } = require('@condo/domains/notification/utils/serverSchema')
+const { BILLING_PAYMENT_FAILED_CATEGORY_TYPE, BILLING_PAYMENT_FAILED_TYPE } = require('@condo/domains/notification/constants/constants')
+
+const { ServiceConsumer } = require('@condo/domains/resident/utils/serverSchema')
+
+const sendPaymentNotifications = async (requestData) => {
+    const { operation, existingItem, updatedItem, context } = requestData
+    const isUpdateOperation =  operation === 'update'
+
+    /**
+     * status changed to PAYMENT_ERROR_STATUS
+     */
+    if (isUpdateOperation && existingItem.status !== PAYMENT_ERROR_STATUS && updatedItem.status === PAYMENT_ERROR_STATUS ) {
+        const serviceConsumer = await ServiceConsumer.getOne(context, { accountNumber: updatedItem.accountNumber })
```

Comment:
We always have mote than one servce consumer for one accountNumber

Reply:
fixed